src/main/java/clients/symphony/api/APIClient.java

```diff
@@ -19,7 +22,14 @@ protected void handleError(Response response, ISymClient botClient) throws SymCl
             );
             throw new ServerErrorException(response.getStatusInfo().getReasonPhrase());
         } else {
-            ClientError error = response.readEntity(ClientError.class);
+            String errorString = response.readEntity(String.class);
+            ClientError error;
+            try {
+                error = mapper.readValue(errorString, ClientError.class);
+            } catch (JsonProcessingException e) {
+                throw new APIClientErrorException(e.getMessage());
```

Comment:
Do we really want to return the `JsonProcessingException` error message or directly `errorString` value? For some raisons it would be interesting to know what has actually been returned by the server, and this error might not be a Json object (it happens with SBE )